index.php

```diff
@@ -7,6 +7,11 @@
 	height: 100%;
 	text-align: center;
 }
+@media (max-width: 500px) {
+  img {
```

Comment:
Do we actually want to affect all `img` tags or just the current on in the `#main` div? I'm just thinking, what if we want to add more images in the future, and those hypothetical future images are smaller

Reply:
Obviously we need to support future images, I don't know what I was thinking

Reply:
Thank you. Looks so much better